### PR TITLE
Fixed issue related to 3893 -- (Min) and "Experienced" (Max) salary d…

### PR DESCRIPTION
**Pull request by vallabha24**
…etails appear to be switched and are shown in the incorrect order

### CODE REVIEW

File: DFC.Digital/DFC.Digital.TLS1.2PerformancePlugin/DFC.Digital.TLS12PerformancePlugin.csproj [hunk 1]
```diff
@@ -31,11 +31,13 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.VisualStudio.QualityTools.WebTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\PublicAssemblies\Microsoft.VisualStudio.QualityTools.WebTestFramework.dll</HintPath>
+      <HintPath>..\packages\Microsoft.VisualStudio.QualityTools.WebTestFramework.12.0.40629.1\lib\net40\Microsoft.VisualStudio.QualityTools.WebTestFramework.dll</HintPath>
```
Comment (trevorkapswarah) [inline DFC.Digital/DFC.Digital.TLS1.2PerformancePlugin/DFC.Digital.TLS12PerformancePlugin.csproj]: @Ilambat does this affect this project?

Comment (Muthuramana) [reply DFC.Digital/DFC.Digital.TLS1.2PerformancePlugin/DFC.Digital.TLS12PerformancePlugin.csproj]: i think using nuget is the correct approach for build servers and dev machines, only question is about the owasp test machine, I guess we should try to fix the func test machine if it causes an issue.

Comment (Muthuramana) [reply DFC.Digital/DFC.Digital.TLS1.2PerformancePlugin/DFC.Digital.TLS12PerformancePlugin.csproj]: I believe this plugin is only used in load test, so owasp func test may not have any issues.

Comment (vallabha24) [reply DFC.Digital/DFC.Digital.TLS1.2PerformancePlugin/DFC.Digital.TLS12PerformancePlugin.csproj]: Thanks for clarifying Muthu

**Pull request merged**